frontend/src/actions/checklists.js

```diff
@@ -66,7 +66,7 @@ export function updateList({ listId, name, description }) {
   return function(dispatch) {
     const meta = { listId }
     dispatch({ type: UPDATE_LIST_REQUEST })
-    AmplifyApi.put('checklists', `/checklist/${listId}`, {
+    AmplifyApi.put('slic-lists-api', `/checklist/${listId}`, {
```

Comment:
The way we do APIs now and how we handle that in the client too is a good blog post subject